content/pages/sponsorship.md

```diff
@@ -0,0 +1,157 @@
+1. __Operations__ ($54/month, $645/annually): Operations include necessary technical items such as domain renewal, web hosting, etc. We have also included general liability insurance, which would protect the A11yTalks project from claims like libel or slander. *These costs are billed annually.*
+
+## How to give
+
+Your financial contributions are tax deductible because A11yTalks is fiscally hosted by a non-profit, ​Open Collective Foundation 501(c)(3).
+
+### Individuals: Join the A11yCat Club
+
+You can set-up a [Monthly Recurring Donation](https://opencollective.com//a11ytalks/contribute/a11ycat-club-36522) or a [One-time Donation](https://opencollective.com//a11ytalks/contribute/one-time-donation-47987) through our Open Collective page. Each level has some contributor benefits as our way of saying “Thanks for being a part of A11yTalks!”
+To receive your A11yCat T-shirt and/or sticker, please follow the instructions on your donation confirmation email from Open Collective. These will be mailed out in monthly batches. Email us at “team [at] a11ytalks [dot] com” if you think we’ve missed you. We promise it wasn’t on purpose.
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4">Join the A11yCat Club (Monthly Recurring)</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td></td>
+  <td>$5/month<br/>($60/year, $0.16/day)</td>
+  <td>$15/month<br/>($180/year, $0.50/day)</td>
+  <td>$25/month<br/>($300/year, $0.83/day)</td>
+</tr>
+<tr>
+  <td class="left">Name listed as a donor on website</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">Name on slide at the beginning of<br/>each A11yTalk</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">A11yCat sticker</td>
+  <td>1</td>
+  <td>2</td>
+  <td>3</td>
+</tr>
+<tr>
+  <td class="left">A11yCat T-shirt</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+</tbody>
+</table>
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4">Join the A11yCat Club (One-Time Donation)</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td></td>
+  <td>$25 donation</td>
+  <td>$100 donation</td>
+  <td>$250 donation</td>
+</tr>
+<tr>
+  <td class="left">Name listed as a donor on website</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+<tr>
+  <td class="left">A11yCat sticker</td>
+  <td>1</td>
+  <td>2</td>
+  <td>3</td>
+</tr>
+<tr>
+  <td class="left">A11yCat T-shirt</td>
+  <td>X</td>
+  <td>x</td>
+  <td>x</td>
+</tr>
+</tbody>
+</table>
+
+### Organizations: Contact us
+
+We prefer to work with organizations who have a positive relationship with the web community and align with our [mission and values](/pages/mission). Please contact us if you are interested in supporting our project.
```

Comment:
- Change "with organizations who have a positive" to "with organizations that have a positive"